llama_stack/providers/tests/inference/test_inference.py

```diff
@@ -132,7 +132,7 @@ async def test_completion(inference_settings):
     params = inference_settings["common_params"]
 
     provider = inference_impl.routing_table.get_provider_impl(params["model"])
-    if provider.__provider_id__ != "meta-reference":
+    if provider.__provider_id__ not in ("meta-reference", "test-ollama"):
```

Comment:
ah my bad. this check should be based on `__provider_type__` and not `__provider_id__` 

can you change it to:

```
if provider.__provider_type__ not in ("meta-reference", "remote::ollama"):
```

Reply:
actually `provider.__provider_type__` is not quite right. hold on.

Reply:
Should be `provider.__provider_spec__.provider_type`

Reply:
done!